django/db/migrations/autodetector.py

```diff
@@ -824,7 +824,7 @@ def generate_renamed_fields(self):
         for app_label, model_name, field_name in sorted(self.new_field_keys - self.old_field_keys):
             old_model_name = self.renamed_models.get((app_label, model_name), model_name)
             old_model_state = self.from_state.models[app_label, old_model_name]
-            new_model_state = self.to_state.models[app_label, old_model_name]
+            new_model_state = self.to_state.models[app_label, model_name]
```

Comment:
The changes seem correct but could you provide a test case for this error?

I assume this only makes a difference when renaming a field on a renamed model?

Reply:
1.  create model (new table)
2.  run makemigrations
3.  do a migration
4.  rename model
5.  run makemigrations

I've got a traceback in step 5.